datasets.ini

```diff
@@ -0,0 +1,13 @@
+[uwsgi]
+module = wsgi:application
+chdir = /home/dnaidoo/Documents/datasets_service
```

Comment:
this path might need a change to make it universal

Reply:
Turns out uwsgi has some magic variables that can be used. Changed both the chdir and socket paths to use a PWD command.